tests/test_ml_classifier.py

```diff
@@ -6,9 +7,45 @@
 from onnx.helper import make_node
 from scipy.sparse import csr_matrix
 
+from src.python_src.util import app_utilities
+from src.python_src.util.app_utilities import app_config, model_file, vectorizer_file
```

Comment:
```suggestion
from src.python_src.util.app_utilities import app_config
```
i might've overlooked it, but i don't think `model_file` or `vectorizer_file` need to be imported